src/Compilers/VisualBasic/Portable/Symbols/BaseTypeAnalysis.vb

```diff
@@ -190,7 +190,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
 
             Debug.Assert(root.OriginalDefinition = root, "root must not be a substitution")
 
-            If current Is Nothing Then
+            If current Is Nothing Or TypeOf current Is MissingMetadataTypeSymbol Then
```

Comment:
We should also see if it is possible to harden the code where the crash happens, where we assume to have at least one location. If that is covered, we might not need this change.